tutorials/plot_eeg_no_mri.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+"""
+EEG source reconstruction with a template MRI
+=============================================
+
+This tutorial explains how to perform source reconstruction using
```

Comment:
I'd also change the title and description a bit to highlight that the tutorial shows how to prepare all the necessary components for source reconstruction but does not actually perform reconstruction.